restapi/src/main/java/io/stargate/web/docsapi/service/query/predicate/AnyValuePredicate.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.stargate.web.docsapi.service.query.predicate;
+
+/**
+ * Special type of predicate that can test a generic value against all database values.
+ *
+ * @param <V> Type of a value used in the predicate.
+ */
+public interface AnyValuePredicate<V> extends StringFilterPredicate<V>, DoubleFilterPredicate<V>, BooleanValuePredicate<V> {
```

Comment:
nit: rename to `ValuePredicate`? "Any" in this context sounds a bit like a predicate that _allows_ any value? WDYT?

Reply:
Yes it will be something like `ValueFilterOperation` or smth like that at the end. Fine?